### DISCUSSION TITLE
Is discord link broken?

### DISCUSSION THREAD

**Discussion by lucario387** [NvChad/NvChad#1870] [Q&A]
Can't accept invite :')
![image](https://user-images.githubusercontent.com/29790821/228792290-f58e41de-cab1-4221-9471-e6941881e3ff.png)

**Comment by siduck** [comment]
@lucario387 I dont think so

**Reply by siduck** [reply]
did you leave the group or someone kicked you out?

**Reply by lucario387** [reply]
Woke up today and I pressed some stuffs while half drunk so I think I left
